modules/persistent-storage-csi-cloning-provisioning.adoc

```diff
@@ -43,7 +41,7 @@ spec:
     name: pvc-1
 ----
 +
-<1> Name of the storage class that provisions the storage back end. Default storage class can be used and `storageClassName` can be omitted in the spec.
+<1> Name of the storage class that provisions the storage back-end. The default storage class can be used and `storageClassName` can be omitted in the spec.
```

Comment:
Same here, back end was correct.

Reply:
Also if you're changing the second sentence to start with "The", you should do the same for the first sentence, "The name of..."